react-components/jsx.md

```diff
@@ -264,20 +281,34 @@ For example:
 
 ```js
 function App() {
+  const [count, setCount] = useState(0)
+
   return (
-    <div className="App">
-      {/* Creates a header tag */}
-      <header className="App-header">
-        {/* Add in the logo */}
-        <img src={logo} className="App-logo" alt="logo" />
+    <>
+      <div>
+        {/* Links for users to learn more about Vite and React */}
```

Comment:
In the text above, should we make it clear that we are adding these comments to the template. Readers could be confused that they're not in the code that vite generated.